torch/_dynamo/debug_utils.py

```diff
@@ -583,6 +583,7 @@ def symint(self, val):
 class InputWriter:
     def __init__(self, save_dir, *, stable_hash=False):
         self._lines = []
+        self._storage_names = []
```

Comment:
unused?

Reply:
Used, outside the class. I'll change it to be accessed by func or at least not private.